src/tests/ftest/verify_perms.py

```diff
@@ -290,10 +290,7 @@ def _real_x(entry_type, path):
 
     '''
     if entry_type == 'file':
-        try:
-            return run_local(logger, path, check=True, verbose=False).returncode == 0
-        except RunException:
-            return False
+        return run_local(logger, path, verbose=False).passed
```

Comment:
I do wonder if the new definition of `run_local` is slower than the older, which might explain why the mu_perms test times out now.
In this daily testing run, the same command took 1min 28s, with a 2 min timeout
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/daily-testing/214/artifact/Functional%20on%20EL%208/dfuse/mu_perms.py/job.log

Reply:
That said, maybe this should just use
```
try:
    return subprocess.run(path, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True).returncode == 0
except subprocess.CalledProcessError:
    return False
```

Reply:
It could be VM slowness like https://daosio.atlassian.net/browse/DAOS-15402 where in:
```
2024-07-18 20:04:36,287 test             L0530 INFO | START 1-./dfuse/mu_perms.py:DfuseMUPerms.test_dfuse_mu_perms;run-container-dfuse-dfuse_with_caching-hosts-pool-server_config-engines-0-storage-0-test_dfuse_mu_perms_cache-verify_perms-5a8a
2024-07-18 20:04:37,835 test             L0473 INFO | ==> Step 1: setUp(): Starting servers [elapsed since last step: 1.55s]
2024-07-18 20:05:40,815 test             L0473 INFO | ==> Step 2: setUp(): Starting agents [elapsed since last step: 62.98s]
2024-07-18 20:06:01,165 test             L0473 INFO | ==> Step 3: setUp(): Setup complete [elapsed since last step: 20.35s]
2024-07-18 20:06:12,590 test             L0473 INFO | ==> Step 4: Verifying simple file permissions on /tmp/daos_dfuse_test_dfuse_mu_perms_1/test_file [elapsed since last step: 11.43s]
2024-07-18 20:06:39,888 test             L0473 INFO | ==> Step 5: Verifying simple dir permissions on /tmp/daos_dfuse_test_dfuse_mu_perms_1/test_dir [elapsed since last step: 27.30s]
2024-07-18 20:07:07,161 test             L0473 INFO | ==> Step 6: Verifying real file permissions on /tmp/daos_dfuse_test_dfuse_mu_perms_1/test_file [elapsed since last step: 27.27s]
2024-07-18 20:09:01,790 test             L0473 INFO | ==> Step 7: Verifying real dir permissions on /tmp/daos_dfuse_test_dfuse_mu_perms_1/test_dir [elapsed since last step: 114.63s]
2024-07-18 20:09:40,306 test             L0473 INFO | ==> Step 8: Creating directory: %s [elapsed since last step: 38.52s]
2024-07-18 20:09:40,704 test             L0473 INFO | ==> Step 9: Giving ownership to daos_test_user_x2 [elapsed since last step: 0.40s]
2024-07-18 20:09:41,456 test             L0473 INFO | ==> Step 10: Verifying real file permissions on /tmp/daos_dfuse_test_dfuse_mu_perms_1/dir1/test_file [elapsed since last step: 0.75s]
2024-07-18 20:11:41,625 test             L0473 INFO | ==> Step 11: tearDown(): Called after test completion (test timeout: 570s, elapsed: 425.34s, remaining: 144.66s) [elapsed since last step: 120.17s]
2024-07-18 20:11:55,161 test             L0933 ERROR| ERROR 1-./dfuse/mu_perms.py:DfuseMUPerms.test_dfuse_mu_perms;run-container-dfuse-dfuse_with_caching-hosts-pool-server_config-engines-0-storage-0-test_dfuse_mu_perms_cache-verify_perms-5a8a -> CommandFailure: verify_perms.py failed on: wolf-102vm3
```
We see that the starting the servers took 62.98s - longer than the norm.  Maybe we should run the command with a larger or no timeout?

Reply:
The old `run_local()` command did not have a default timeout, but the new implementation - where the method arguments were updated to align with `run_remote()` - implemented a 120 second default timeout.  I'm updating the new `run_local()` to set the default timeout to None for compatibility reasons.